app/submitter/submitter.py

```diff
@@ -39,7 +40,14 @@ def send_message(self, message: str, tx_id: str, case_id: str) -> bool:
 
         # DEFAULT_RETRY is not idempotent.
         # However, this behaviour was deemed acceptable for our use case.
-        blob.upload_from_string(str(message).encode("utf8"), retry=DEFAULT_RETRY)
+        # Treats doubble submission error as successful because GCS doesn't allow partial data to be uploaded.
+        try:
+            blob.upload_from_string(str(message).encode("utf8"), retry=DEFAULT_RETRY)
+        except Forbidden as e:
+            if "storage.objects.delete" in e.message:
+                logger.info("Double Submission found")
```

Comment:
Maybe something like `Questionnaire submission exists, ignoring delete operation error` will be clearer for the log message as double submission is still possible if SDX deletes the object in time, i.e. retry strategy kicks in etc.
Also, worth adding a comment why were are doing this as it is not strictly obvious